docs/application/dotnet/guides/nui/grid-layout.md

```diff
@@ -0,0 +1,41 @@
+# Grid Layout
+
+[Overview](#overview)<br>
+[Column](#column)<br>
```

Comment:
docs.tizen.org automatically generates TOC on the right side. 
Please remove TOC.

Reply:
Okay, Removed TOC.